services/hanke-service/src/main/resources/db/changelog/changesets/create-hankegeometriat-tables.sql

```diff
@@ -0,0 +1,68 @@
+--liquibase formatted sql
+--changeset teemu:create-hankegeometry-tables comment:Create Hanke Geometry tables
+
+CREATE EXTENSION IF NOT EXISTS postgis;
+
+CREATE TABLE IF NOT EXISTS HankeGeometriat
+(
+    id              integer GENERATED BY DEFAULT AS IDENTITY,
+    hankeId         integer,
+    version         integer,
+    createdByUserId integer,
+    createdAt       timestamp,
+    updatedByUserId integer,
```

Comment:
"updated" vs. "modified". This has gone two ways at some point, see:
https://helsinkisolutionoffice.atlassian.net/wiki/spaces/HAI/pages/928809010/Sovittuja+kehitysk+yt+nteit